tasks/css/easy/styles.css

```diff
@@ -1,3 +1,5 @@
 /* CSS - Easy */
-
+ h1{
+    color: red;
+ }
```

Comment:
Looks good!

The task was about creating the .red class but this works too just fine.

Reply:
Really appretiated for providing us with this. Thank you soo much.